schemas/donor.json

```diff
@@ -182,6 +182,29 @@
       "valueType": "integer",
       "meta": { "displayName": "BMI" }
     },
+    {
+      "description": "Indicate presence of any hereditary genetic disorders. (Reference: ORPHA)",
```

Comment:
is this the reference source? https://www.rd-code.eu/introduction/

Reply:
Theoretically i think this looks' fine @hknahal, but the list seems really incomplete (short) if there is a reference code.

Reply:
I used Orphanet (https://www.orpha.net/consor/cgi-bin/index.php) to standardize the genetic disorder names. I wish I could show you an example link, but their website is down for some reason today. When I was reviewing programs case report forms, some were collecting information about genetic disorders, but they used different names (ie. they weren't always consistent). For example, one program just used "MEN-1", while another used "Multiple endocrine neoplasia 1". But in Orphanet, the disease name is "Multiple endocrine neoplasia type 1" which is what I included in the controlled terminology. So maybe I should remove the reference to Orphanet because it is misleading and makes it seem Orphanet has a list of genetic disorders specific to just cancer. 
 
I only included the genetic disorders indicated in the programs' CRFs but I am sure there are more genetic disorders that are applicable to cancer. I can't find standardized list of genetic disorders specific to only cancer, but I'd like to mention somewhere that programs should contact us if they want to include more terms in the genetic_disorders field.

Reply:
ok yes i found those sites but they were down for me too! 
- i think the reference to ORPHA is good to keep, maybe jsut clarify in notes how its being used? 
- maybe include in the `notes`  that a program can contact us if they want to include more terms in the genetic_disorders list is they see something missing?